src/components/ProfileTree/ProfileLabel.js

```diff
@@ -133,10 +147,14 @@ export const ProfileLabel = memo(({
 });
 
 ProfileLabel.propTypes = {
-  entityKey: PropTypes.string.isRequired,
-  recordData: PropTypes.object.isRequired,
   linkingRules: PropTypes.object.isRequired,
+  recordData: PropTypes.object.isRequired,
+  parentRecordData: PropTypes.object.isRequired,
+  parentSectionKey: PropTypes.string.isRequired,
+  parentSectionData: PropTypes.arrayOf(PropTypes.object).isRequired,
+  setParentSectionData: PropTypes.func.isRequired,
   label: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
+  reactTo: PropTypes.string,
```

Comment:
Add this prop to the `defaultProps` if it is not required, please.

Reply:
Done